BotUtils.py

```diff
@@ -182,6 +182,8 @@ def click(self, location: tuple | tuple, amount=1, timeout=0.1, move_timeout=0.1
             """
             if amount > 1:
                 time.sleep(timeout)
```

Comment:
will readd this and try it out, commented it out since last PR cause something wasn't working properly

Reply:
Do you remember what was not working properly?